docs/examples/DataSet/Data acquisition with help of threads .ipynb

```diff
@@ -0,0 +1,602 @@
+{
+ "cells": [
```

Comment:
Remove trailing spaces in file name. I would also call it either `Data acquisition with the help of threads.ipynb` or perhaps easier to read simply  `Threaded Data acquisition.ipynb`

Reply:
Ah shoot! missed that space! Will update the title :)